README.md

```diff
@@ -35,183 +27,57 @@ OpenTTD is licensed under the GNU General Public License version 2.0,
-   whether we can quickly reproduce the desync. Otherwise we will need …
-- the first saved savegame (search for the first line beginning with 'save'
-   where the first part, up to the last underscore '_', is the same). We need
-   this savegame to be able to reproduce the bug when the last savegame is not
-   old enough. If you loaded a scenario or savegame you need to attach that.
-- optionally you can attach the savegames from around 50%, 75%, 85%, 90% and
-   95% of the game's progression. We can use these savegames to speed up the
-   reproduction of the desync, but we should be able to reproduce these
-   savegames based on the first savegame and commands-out.log.
-- in case you use any NewGRFs you should attach the ones you used unless
-   we can easily find them ourselves via bananas or when they are in the
-   #openttdcoop pack.
-
-Do NOT remove the dmp_cmds savegames of a desync you have reported until the
-desync has been fixed; if you, by accident, send us the wrong savegames we
-will not be able to reproduce the desync and thus will be unable to fix it.
-
-## 3.0) Supported platforms
-
-OpenTTD has been ported to several platforms and operating systems. It should
-not be very difficult to port it to a new platform. The currently working
-platforms are:
+## 1.1) Downloading OpenTTD
+
+OpenTTD can be downloaded from the [official OpenTTD website](https://www.openttd.org/).
+
+Both 'stable' and 'nightly' versions are available for download.
+
+Most people will want the 'stable' version.
+
+The 'nightly' version includes the latest code changes and may be less reliable.
+
+On some platforms OpenTTD will also be available via your OS package manager or a similar service.
+
+
+## 1.2) OpenTTD gameplay manual
+
+OpenTTD has a [community-maintained wiki](https://wiki.openttd.org/), including a gameplay manual and tips.
+
+
+## 1.3) Supported platforms
+
+OpenTTD has been ported to several platforms and operating systems.
+
+The currently working platforms are:
 
 - FreeBSD (SDL)
 - Haiku (SDL)
-- Linux (SDL or Allegro)
+- Linux (SDL)
 - macOS (universal) (Cocoa video and sound drivers)
 - OpenBSD (SDL)
 - OS/2 (SDL)
-- Windows (Win32 GDI (faster) or SDL or Allegro)
+- Windows (Win32 GDI (faster) or SDL)
 
-## 4.0) Installing and running OpenTTD
+### 1.3.1) Legacy support
+Platforms, languages and compilers change.
+We'll keep support going on old platforms as long as someone is interested in supporting them, except where it means the project can't move the project forward to keep up with language and compiler features.
 
-Installing OpenTTD is fairly straightforward. Either you have downloaded an
-archive which you have to extract to a directory where you want OpenTTD to
-be installed, or you have downloaded an installer, which will automatically
-extract OpenTTD in the given directory.
+We'll also do our best to maintain compatibility with old savegames, except where it becomes impractical.
```

Comment:
This statement is probably wrong, savegame compatibility is of much higher priority

Reply:
* this clause could be rewritten entirely
* or the 'except where it becomes impractical' could simply be dropped

I think that declaring 100% savegame compatibility as an absolute constraint isn't wise, and causes perception problems such as some of https://github.com/OpenTTD/OpenTTD/issues/7602  YMMV etc